netci.groovy

```diff
@@ -30,10 +30,15 @@ platformList.each { platform ->
     def dockerWorkingDirectory = "/src/core-setup"
     def dockerCommand = ''
     def crossbuildargs = ''
+    def buildArgs = "-ConfigurationGroup=${configuration} -TargetArchitecture=${architecture}"
+
+    if (os != 'Windows_NT' && architecture != 'armel' && configuration == 'Release') {
```

Comment:
we're stripping symbols in official arm builds.  My first thought when I saw the failure was to just turn this off for arm builds in CI, but since we're doing this for official arm builds, it's not clear why this is failing.

https://github.com/dotnet/core-setup/blob/master/buildpipeline/pipeline.json#L39